src/main/kotlin/com/jay/racingcar/domain/RacingCar.kt

```diff
@@ -1,10 +1,17 @@
 package com.jay.racingcar.domain
 
 class RacingCar(
+    private var name: Name,
     private val movingStrategy: MovingStrategy,
-    internal var position: Int = 0
-) {
+    private var position: Int = 0
+) : Comparable<RacingCar> {
     fun move() {
         if (movingStrategy.canMove()) position++
     }
+
+    fun getPosition(): Int = position
+    fun getNameValue(): String = name.value
+    override fun compareTo(other: RacingCar): Int {
+        return other.position.compareTo(position)
+    }
```

Comment:
```suggestion
    fun getPosition(): Int = position
    
    fun getNameValue(): String = name.value
    
    override fun compareTo(other: RacingCar): Int {
        return other.position.compareTo(position)
    }
```
함수 단위로 개행을 해주시면, 코드가 좀 더 보기 편해져요. 😄

Reply:
내부에서 변경이 가능하고 외부에서는 접근만 가능하기 위해, get 관련 함수들을 구현해주셨네요.
지금과 같이 구현해주셔도 좋지만, 프로퍼티의 특징을 활용하면 아래와 같이 구현할 수 있습니다.
```kotlin
class RacingCar(
   name: Name
) {
    private var _name = name
    val name get() = _name
}
```

Reply:
리뷰어님은 어떤 스타일을 더 선호하시나요?